.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -58,3 +54,7 @@ If you need to re-run a test, please mark the related checkbox, it will be unche
 - [ ] Re-run test "susemanager_unittests"
 - [ ] Re-run test "javascript_lint"
 - [ ] Re-run test "spacecmd_unittests"
+
+# Before you merge
+
+Check [[How to branch and merge properly]]!
```

Comment:
Are you sure this shows up correctly? Above we use the full url to a wiki page.

Reply:
Yikes, I tested it on the Wiki itself, but of course this doesn't work from a PR. Fixing it.

```suggestion
# Before you merge

Check [How to branch and merge properly](https://github.com/uyuni-project/uyuni/wiki/How-to-branch-and-merge-properly)!
```

Reply:
Fixed, and requesting a new review.